.github/workflows/build-docker-image-and-binaries.yaml

```diff
@@ -1,16 +1,12 @@
-name: Upload Binaries and Docker Image
+name: Build and Upload Release
 
 on:
   workflow_dispatch:
     inputs:
       tag:
         description: Github tag to release binaries for (reusing an existing tag will make the pipeline fail)
         required: true
-        default: latest
-      wasm:
-        description: Whether to enable wasm ("true"/"false")
-        required: true
-        default: "true"
+        default: vX.Y.Z
       ibc-wasm-hooks:
```

Comment:
shall we default this to false? we can fork the workflow in a branch and do it manually when needed